tests/main/snapd-snap/task.yaml

```
@@ -166,19 +174,142 @@ execute: |
     EOF
     chmod +x snapcraft-cleanup.sh
     tests.cleanup defer sh -c "$PWD/snapcraft-cleanup.sh"
-
     echo "Ensure we have the fc-cache binaries"
     unsquashfs -ll snapd_spread-test.snap | MATCH bin/fc-cache-v6
     unsquashfs -ll snapd_spread-test.snap | MATCH bin/fc-cache-v7
-
     echo "Ensure we have xdelta3"
     unsquashfs -ll snapd_spread-test.snap | MATCH bin/xdelta3
-
     echo "Ensure we have mksquashfs (and the dependencies)"
     unsquashfs -ll snapd_spread-test.snap | MATCH bin/mksquashfs
     unsquashfs -ll snapd_spread-test.snap | MATCH liblzma.so.5
-
     echo "Ensure we have ld.so.conf"
     unsquashfs -ll snapd_spread-test.snap | MATCH etc/ld.so.conf
     echo "Ensure we have libc"
     unsquashfs -ll snapd_spread-test.snap | MATCH libc.so
+    echo "Ensure we can install the snapd snap"
+    snap install --dangerous snapd_spread-test.snap
+    cat >> snapd-cleanup.sh <<EOF
```

does this need a little tweak?
```suggestion
    cat >> snapd-cleanup.sh <<'EOF'
```

---

As Pawel suggested this can actually just be a normal file and ignore this problem